common/user.js

```diff
@@ -510,6 +510,69 @@ class User {
     });
   };
 
+  /**
+   * POST - /api/users/csv
+   *
+   * Retrieves a list of users (admin only).
+   * @name /users/csv
+   */
+  static apiGetUsersCSV (req, res, next) {
+    ArkimeUtil.noCache(req, res, 'text/csv');
+
+    if (typeof req.body !== 'object') { return res.send('Invalid'); }
+    if (Array.isArray(req.body.start) || Array.isArray(req.body.length)) {
+      return res.send('Invalid');
+    }
+
+    const query = {
+      from: parseInt(req.body.start) || 0,
+      size: parseInt(req.body.length) || 10000
+    };
+
+    if (ArkimeUtil.isString(req.body.filter)) {
```

Comment:
We do this in `apiGetUsers` too. Should we abstract this to a new function?

Reply:
ok, will do